crates/scroll/evm/src/execute.rs

```diff
@@ -95,10 +91,7 @@ mod tests {
     const EUCLID_V2_BLOCK_NUMBER: u64 = 14907015;
     const EUCLID_V2_BLOCK_TIMESTAMP: u64 = 1745305200;
     const FEYNMAN_BLOCK_TIMESTAMP: u64 = 1755576000;
-
-    const L1_BASE_FEE_SLOT: U256 = U256::from_limbs([1, 0, 0, 0]);
-    const OVER_HEAD_SLOT: U256 = U256::from_limbs([2, 0, 0, 0]);
-    const SCALAR_SLOT: U256 = U256::from_limbs([3, 0, 0, 0]);
+    const GALILEO_BLOCK_TIMESTAMP: u64 = 1755576001; // TODO(thegaram): update to actual timestamp
```

Comment:
Can update this when we set the timestamp for the Galileo mainnet release, in another PR.